compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -1406,6 +1398,21 @@ private void handleDeclaredWithVar(BLangVariable variable) {
         }
     }
 
+    private void handleWildCardBindingVariable(BLangSimpleVariable simpleVariable, BType rhsType) {
+        if (simpleVariable.name.value.equals(Names.IGNORE.value)) {
+            BType expectedType = simpleVariable.getBType();
+            BType actualType = rhsType;
+            if (expectedType != null) {
+                actualType = expectedType;
+            }
+            expectedType = symTable.anyType;
```

Comment:
We can directly use anyType in the subsequent line, right? Shall we reconsider the variable names also?

expectedType -> varType 
actualType -> bindingValueType